apps/frontend/src/pages/analytics/libs/helpers/get-analytics-columns/get-analytics-columns.helper.tsx

```diff
@@ -0,0 +1,35 @@
+import { type TableColumn } from "~/libs/types/types.js";
+
+import { type AnalyticsRow } from "../../types/types.js";
+
+const getAnalyticsColumns = (
+	dateRange: string[],
+): TableColumn<AnalyticsRow>[] => {
+	const columns: TableColumn<AnalyticsRow>[] = [
+		{
+			accessorKey: "contributorName",
+			header: "Name",
+			size: 220,
+		},
+	];
+
+	const dateColumns: TableColumn<AnalyticsRow>[] = dateRange.map((date) => ({
+		accessorKey: `commitsNumber_${date}.commitsNumber`,
+		cell: ({ getValue }): JSX.Element => {
```

Comment:
```suggestion
		cell: ({ getValue })=> {
```
just according to other code